app/views/layouts/application.html.slim

```diff
@@ -10,16 +11,11 @@ html
     link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF" crossorigin="anonymous"
     script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-kQtW33rZJAHjgefvhyyzcGF3C5TFyBQBA13V1RKPf4uH+bwyzQxZ6CmMZHmNBEfJ" crossorigin="anonymous"
 
-  body
+  / TODO: Move styles
+  body style="margin-bottom: 140px;"
     nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between"
       a href="#{root_path}" class="navbar-brand"
-        | My Own Blog
-
-      ul class="navbar-nav"
-        li class="nav-item"
-          = link_to "About me", pages_about_path, class: "nav-link #{yield :pages_about_path}"
-        li class="nav-item"
-          = link_to "Contact me", new_feedback_path, class: "nav-link #{yield :new_feedback_path}"
+        = current_user&.companies&.first&.official_name || "My Own Blog"
```

Comment:
Can a user be a member of two or more companies?

Reply:
Yeah, for sure. I need to think about this moment. :)